src/views/ReadArticle.js

```diff
@@ -305,8 +309,24 @@ export class Article extends Component {
                 <img className="share-icon" src={thumbsUp} alt="logo" />
                 <img className="share-icon" src={dislikeIcon} alt="logo" />
                 <img className="share-icon" src={bookmarkIcon} alt="logo" />
-                <img className="share-icon" src={facebookIcon} alt="logo" />
-                <img className="share-icon" src={twitterIcon} alt="logo" />
+                <a
+                  className="share-icon"
+                  href={`https://www.facebook.com/sharer/sharer.php?&u=${currentUrl}`}
+                >
+                  <img className="share-icon" src={facebookIcon} alt="logo" />
+                </a>
+                <a
+                  className="share-icon"
+                  href={`http://twitter.com/share?url=${currentUrl}&text=Hello world, also found this relevant&hashtags=authorsHeaven,software development`}
+                >
+                  <img className="share-icon" src={twitterIcon} alt="logo" />
+                </a>
+                <a
+                  className="share-icon"
+                  href={`mailto:?subject=Sharing the inspiring article&body=${currentUrl}`}
+                >
```

Comment:
@Musigwa is it possible to refactor the body of this email? I mean this body should be an anchor tag with the link to the article, in a way that a user can click it. Now a user can see a plain link only.

Reply:
Yes, thanks I'll try to do that to make it better.